app/models/user.rb

```diff
@@ -8,4 +8,9 @@ class User < ActiveRecord::Base
   validates_presence_of :company_id
 
   mount_uploader :profile_image, ProfileImageUploader
+
+  
+  def get_all_users_for_company(company_id)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)